docs/content/1.getting-started/3.theme.md

```diff
@@ -746,6 +746,7 @@ This is how the `@theme` is generated for each design token:
   --outline-color-default: var(--ui-border);
   --outline-color-inverted: var(--ui-border-inverted);
   --stroke-color-default: var(--ui-border);
```

Comment:
If you need this one, let's add them all for `stroke` and `fill`. So you can add `--stroke-color-muted`, `--fill-color-muted` and `--fill-color-accented` I guess.